Race condition in release workflow

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Expected behavior

Release workflow releases

### Current behavior

Release workflow breaks: https://github.com/neurobagel/query-tool/actions/runs/18985634501/job/54228525327

<details>

<summary>Logfile excerpt</summary>

Shortened irrelevant stuff (getting labels and commits)

```
﻿2025-10-31T21:26:23.1166510Z Current runner version: '2.329.0'
2025-10-31T21:26:23.1192319Z ##[group]Runner Image Provisioner
2025-10-31T21:26:23.1193373Z Hosted Compute Agent
2025-10-31T21:26:23.1194142Z Version: 20251016.436
2025-10-31T21:26:23.1194925Z Commit: 8ab8ac8bfd662a3739dab9fe09456aba92132568
2025-10-31T21:26:23.1195839Z Build Date: 2025-10-15T20:44:12Z
2025-10-31T21:26:23.1196640Z ##[endgroup]
2025-10-31T21:26:23.1197383Z ##[group]Operating System
2025-10-31T21:26:23.1198167Z Ubuntu
2025-10-31T21:26:23.1198834Z 24.04.3
2025-10-31T21:26:23.1199707Z LTS
2025-10-31T21:26:23.1200351Z ##[endgroup]
2025-10-31T21:26:23.1201062Z ##[group]Runner Image
2025-10-31T21:26:23.1201751Z Image: ubuntu-24.04
2025-10-31T21:26:23.1202462Z Version: 20251030.96.2
2025-10-31T21:26:23.1203653Z Included Software: https://github.com/actions/runner-images/blob/ubuntu24/20251030.96/images/ubuntu/Ubuntu2404-Readme.md
2025-10-31T21:26:23.1205337Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu24%2F20251030.96
2025-10-31T21:26:23.1206454Z ##[endgroup]
2025-10-31T21:26:23.1209042Z ##[group]GITHUB_TOKEN Permissions
2025-10-31T21:26:23.1211512Z Actions: write
2025-10-31T21:26:23.1212265Z Attestations: write
2025-10-31T21:26:23.1212916Z Checks: write
2025-10-31T21:26:23.1213600Z Contents: write
2025-10-31T21:26:23.1214295Z Deployments: write
2025-10-31T21:26:23.1214949Z Discussions: write
2025-10-31T21:26:23.1215606Z Issues: write
2025-10-31T21:26:23.1216302Z Metadata: read
2025-10-31T21:26:23.1216978Z Models: read
2025-10-31T21:26:23.1217610Z Packages: write
2025-10-31T21:26:23.1218320Z Pages: write
2025-10-31T21:26:23.1218971Z PullRequests: write
2025-10-31T21:26:23.1219803Z RepositoryProjects: write
2025-10-31T21:26:23.1220664Z SecurityEvents: write
2025-10-31T21:26:23.1221473Z Statuses: write
2025-10-31T21:26:23.1222137Z ##[endgroup]
2025-10-31T21:26:23.1224349Z Secret source: Actions
2025-10-31T21:26:23.1225253Z Prepare workflow directory
2025-10-31T21:26:23.1534303Z Prepare all required actions
2025-10-31T21:26:23.1573646Z Getting action download info
2025-10-31T21:26:23.6238223Z Download action repository 'actions/create-github-app-token@v2' (SHA:67018539274d69449ef7c02e8e71183d1719ab42)
2025-10-31T21:26:24.4453216Z Download action repository 'actions/checkout@v5' (SHA:08c6903cd8c0fde910a37f88322edcfb5dd907a8)
2025-10-31T21:26:24.5730359Z Complete job name: auto-release
2025-10-31T21:26:24.6714737Z ##[group]Run actions/create-github-app-token@v2
2025-10-31T21:26:24.6716535Z with:
2025-10-31T21:26:24.6717663Z   app-id: 949602
2025-10-31T21:26:24.6743823Z   private-key: ***

2025-10-31T21:26:24.6745084Z   skip-token-revoke: false
2025-10-31T21:26:24.6746480Z   github-api-url: https://api.github.com
2025-10-31T21:26:24.6748246Z ##[endgroup]
2025-10-31T21:26:24.7755867Z Inputs 'owner' and 'repositories' are not set. Creating token for this repository (neurobagel/query-tool).
2025-10-31T21:26:25.3084411Z ##[group]Run actions/checkout@v5
2025-10-31T21:26:25.3085667Z with:
2025-10-31T21:26:25.3086662Z   fetch-depth: 0
2025-10-31T21:26:25.3088151Z   token: ***
2025-10-31T21:26:25.3089263Z   repository: neurobagel/query-tool
2025-10-31T21:26:25.3090865Z   ssh-strict: true
2025-10-31T21:26:25.3091937Z   ssh-user: git
2025-10-31T21:26:25.3093011Z   persist-credentials: true
2025-10-31T21:26:25.3094170Z   clean: true
2025-10-31T21:26:25.3095288Z   sparse-checkout-cone-mode: true
2025-10-31T21:26:25.3096537Z   fetch-tags: false
2025-10-31T21:26:25.3097619Z   show-progress: true
2025-10-31T21:26:25.3098770Z   lfs: false
2025-10-31T21:26:25.3099954Z   submodules: false
2025-10-31T21:26:25.3101077Z   set-safe-directory: true
2025-10-31T21:26:25.3102228Z ##[endgroup]
2025-10-31T21:26:25.3932320Z Syncing repository: neurobagel/query-tool
2025-10-31T21:26:25.3936809Z ##[group]Getting Git version info
2025-10-31T21:26:25.3938439Z Working directory is '/home/runner/work/query-tool/query-tool'
2025-10-31T21:26:25.3940973Z [command]/usr/bin/git version
2025-10-31T21:26:25.3981387Z git version 2.51.2
2025-10-31T21:26:25.4000080Z ##[endgroup]
2025-10-31T21:26:25.4011839Z Temporarily overriding HOME='/home/runner/work/_temp/e6a33412-3629-4b62-b617-5aa310a96083' before making global git config changes
2025-10-31T21:26:25.4015268Z Adding repository directory to the temporary git global config as a safe directory
2025-10-31T21:26:25.4017969Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/query-tool/query-tool
2025-10-31T21:26:25.4045975Z Deleting the contents of '/home/runner/work/query-tool/query-tool'
2025-10-31T21:26:25.4049073Z ##[group]Initializing the repository
2025-10-31T21:26:25.4052763Z [command]/usr/bin/git init /home/runner/work/query-tool/query-tool
2025-10-31T21:26:25.4134557Z hint: Using 'master' as the name for the initial branch. This default branch name
2025-10-31T21:26:25.4138127Z hint: is subject to change. To configure the initial branch name to use in all
2025-10-31T21:26:25.4140573Z hint: of your new repositories, which will suppress this warning, call:
2025-10-31T21:26:25.4142623Z hint:
2025-10-31T21:26:25.4143853Z hint: 	git config --global init.defaultBranch <name>
2025-10-31T21:26:25.4146013Z hint:
2025-10-31T21:26:25.4147377Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2025-10-31T21:26:25.4149393Z hint: 'development'. The just-created branch can be renamed via this command:
2025-10-31T21:26:25.4151245Z hint:
2025-10-31T21:26:25.4152314Z hint: 	git branch -m <name>
2025-10-31T21:26:25.4153472Z hint:
2025-10-31T21:26:25.4154919Z hint: Disable this message with "git config set advice.defaultBranchName false"
2025-10-31T21:26:25.4157064Z Initialized empty Git repository in /home/runner/work/query-tool/query-tool/.git/
2025-10-31T21:26:25.4160834Z [command]/usr/bin/git remote add origin https://github.com/neurobagel/query-tool
2025-10-31T21:26:25.4177050Z ##[endgroup]
2025-10-31T21:26:25.4180421Z ##[group]Disabling automatic garbage collection
2025-10-31T21:26:25.4183059Z [command]/usr/bin/git config --local gc.auto 0
2025-10-31T21:26:25.4203717Z ##[endgroup]
2025-10-31T21:26:25.4206712Z ##[group]Setting up auth
2025-10-31T21:26:25.4209936Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2025-10-31T21:26:25.4236705Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2025-10-31T21:26:25.4557324Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2025-10-31T21:26:25.4583471Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2025-10-31T21:26:25.4747502Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2025-10-31T21:26:25.4774034Z ##[endgroup]
2025-10-31T21:26:25.4775814Z ##[group]Fetching the repository
2025-10-31T21:26:25.4783034Z [command]/usr/bin/git -c protocol.version=2 fetch --prune --no-recurse-submodules origin +refs/heads/*:refs/remotes/origin/* +refs/tags/*:refs/tags/*
2025-10-31T21:26:26.6244042Z From https://github.com/neurobagel/query-tool
2025-10-31T21:26:26.6245263Z  * [new branch]      main           -> origin/main
2025-10-31T21:26:26.6246252Z  * [new tag]         v0.1.0         -> v0.1.0
2025-10-31T21:26:26.6247120Z  * [new tag]         v0.10.0        -> v0.10.0
2025-10-31T21:26:26.6247964Z  * [new tag]         v0.10.1        -> v0.10.1
2025-10-31T21:26:26.6248933Z  * [new tag]         v0.10.2-next.0 -> v0.10.2-next.0
2025-10-31T21:26:26.6250206Z  * [new tag]         v0.11.0        -> v0.11.0
2025-10-31T21:26:26.6251130Z  * [new tag]         v0.11.1        -> v0.11.1
2025-10-31T21:26:26.6252023Z  * [new tag]         v0.12.0        -> v0.12.0
2025-10-31T21:26:26.6253014Z  * [new tag]         v0.2.0         -> v0.2.0
2025-10-31T21:26:26.6253954Z  * [new tag]         v0.2.1         -> v0.2.1
2025-10-31T21:26:26.6254894Z  * [new tag]         v0.2.2         -> v0.2.2
2025-10-31T21:26:26.6256116Z  * [new tag]         v0.3.0         -> v0.3.0
2025-10-31T21:26:26.6257037Z  * [new tag]         v0.4.0         -> v0.4.0
2025-10-31T21:26:26.6257882Z  * [new tag]         v0.5.0         -> v0.5.0
2025-10-31T21:26:26.6258835Z  * [new tag]         v0.6.0         -> v0.6.0
2025-10-31T21:26:26.6259843Z  * [new tag]         v0.7.0         -> v0.7.0
2025-10-31T21:26:26.6260629Z  * [new tag]         v0.7.1         -> v0.7.1
2025-10-31T21:26:26.6261397Z  * [new tag]         v0.7.2         -> v0.7.2
2025-10-31T21:26:26.6262165Z  * [new tag]         v0.8.0         -> v0.8.0
2025-10-31T21:26:26.6262950Z  * [new tag]         v0.8.1         -> v0.8.1
2025-10-31T21:26:26.6263617Z  * [new tag]         v0.9.0         -> v0.9.0
2025-10-31T21:26:26.6264295Z  * [new tag]         v0.9.1         -> v0.9.1
2025-10-31T21:26:26.6265020Z  * [new tag]         v0.9.2         -> v0.9.2
2025-10-31T21:26:26.6265594Z  * [new tag]         v0.9.3         -> v0.9.3
2025-10-31T21:26:26.6266012Z  * [new tag]         v0.9.4         -> v0.9.4
2025-10-31T21:26:26.6266470Z  * [new tag]         v0.9.5         -> v0.9.5
2025-10-31T21:26:26.6294035Z [command]/usr/bin/git branch --list --remote origin/main
2025-10-31T21:26:26.6314027Z   origin/main
2025-10-31T21:26:26.6320891Z [command]/usr/bin/git rev-parse refs/remotes/origin/main
2025-10-31T21:26:26.6337768Z e0b7dba85e7a326c8ff4219cfeb526af9401c956
2025-10-31T21:26:26.6341500Z ##[endgroup]
2025-10-31T21:26:26.6342430Z ##[group]Determining the checkout info
2025-10-31T21:26:26.6343521Z ##[endgroup]
2025-10-31T21:26:26.6347850Z [command]/usr/bin/git sparse-checkout disable
2025-10-31T21:26:26.6382804Z [command]/usr/bin/git config --local --unset-all extensions.worktreeConfig
2025-10-31T21:26:26.6402270Z ##[group]Checking out the ref
2025-10-31T21:26:26.6406293Z [command]/usr/bin/git checkout --progress --force -B main refs/remotes/origin/main
2025-10-31T21:26:26.6498089Z Switched to a new branch 'main'
2025-10-31T21:26:26.6499039Z branch 'main' set up to track 'origin/main'.
2025-10-31T21:26:26.6504186Z ##[endgroup]
2025-10-31T21:26:26.6573277Z [command]/usr/bin/git log -1 --format=%H
2025-10-31T21:26:26.6591004Z e0b7dba85e7a326c8ff4219cfeb526af9401c956
2025-10-31T21:26:26.6698970Z ##[group]Run auto_download_url="$(curl -fsSL https://api.github.com/repos/intuit/auto/releases/latest | jq -r '.assets[] | select(.name == "auto-linux.gz") | .browser_download_url')"
2025-10-31T21:26:26.6700594Z [36;1mauto_download_url="$(curl -fsSL https://api.github.com/repos/intuit/auto/releases/latest | jq -r '.assets[] | select(.name == "auto-linux.gz") | .browser_download_url')"[0m
2025-10-31T21:26:26.6701442Z [36;1mwget -O- "$auto_download_url" | gunzip > ~/auto[0m
2025-10-31T21:26:26.6701877Z [36;1mchmod a+x ~/auto[0m
2025-10-31T21:26:26.6726992Z shell: /usr/bin/bash -e {0}
2025-10-31T21:26:26.6727408Z ##[endgroup]
2025-10-31T21:26:26.9496250Z --2025-10-31 21:26:26--  https://github.com/intuit/auto/releases/download/v11.3.0/auto-linux.gz
2025-10-31T21:26:26.9751801Z Resolving github.com (github.com)... 140.82.113.4
2025-10-31T21:26:27.0332473Z Connecting to github.com (github.com)|140.82.113.4|:443... connected.
2025-10-31T21:26:27.2907404Z HTTP request sent, awaiting response... 302 Found
2025-10-31T21:26:27.2915497Z Location: https://release-assets.githubusercontent.com/github-production-release-asset/160233598/be0f9919-803d-409b-a8be-9f66a4fb2f2e?sp=r&sv=2018-11-09&sr=b&spr=https&se=2025-10-31T22%3A03%3A03Z&rscd=attachment%3B+filename%3Dauto-linux.gz&rsct=application%2Foctet-stream&skoid=96c2d410-5711-43a1-aedd-ab1947aa7ab0&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skt=2025-10-31T21%3A02%3A15Z&ske=2025-10-31T22%3A03%3A03Z&sks=b&skv=2018-11-09&sig=nK99UhCSyILUS2lXD%2BZYXh03b1xFf31vFFJm%2BDUHXtM%3D&jwt=*** [following]
2025-10-31T21:26:27.2925993Z --2025-10-31 21:26:27--  https://release-assets.githubusercontent.com/github-production-release-asset/160233598/be0f9919-803d-409b-a8be-9f66a4fb2f2e?sp=r&sv=2018-11-09&sr=b&spr=https&se=2025-10-31T22%3A03%3A03Z&rscd=attachment%3B+filename%3Dauto-linux.gz&rsct=application%2Foctet-stream&skoid=96c2d410-5711-43a1-aedd-ab1947aa7ab0&sktid=398a6654-997b-47e9-b12b-9515b896b4de&skt=2025-10-31T21%3A02%3A15Z&ske=2025-10-31T22%3A03%3A03Z&sks=b&skv=2018-11-09&sig=nK99UhCSyILUS2lXD%2BZYXh03b1xFf31vFFJm%2BDUHXtM%3D&jwt=***
2025-10-31T21:26:27.2955639Z Resolving release-assets.githubusercontent.com (release-assets.githubusercontent.com)... 185.199.109.133, 185.199.110.133, 185.199.111.133, ...
2025-10-31T21:26:27.3038218Z Connecting to release-assets.githubusercontent.com (release-assets.githubusercontent.com)|185.199.109.133|:443... connected.
2025-10-31T21:26:27.3944411Z HTTP request sent, awaiting response... 200 OK
2025-10-31T21:26:27.3945272Z Length: 46266273 (44M) [application/octet-stream]
2025-10-31T21:26:27.3946201Z Saving to: ‘STDOUT’
2025-10-31T21:26:27.3949749Z 
2025-10-31T21:26:27.4043217Z      0K .......... .......... .......... .......... ..........  0% 5.01M 9s
\2025-10-31T21:26:28.4112213Z  44900K .......... .......... .......... .......... .......... 99%  540M 0s
2025-10-31T21:26:28.4112995Z  44950K .......... .......... .......... .......... .......... 99%  689M 0s
2025-10-31T21:26:28.4113781Z  45000K .......... .......... .......... .......... .......... 99%  831M 0s
2025-10-31T21:26:28.4114582Z  45050K .......... .......... .......... .......... .......... 99%  513M 0s
2025-10-31T21:26:28.4174911Z  45100K .......... .......... .......... .......... .......... 99% 8.01M 0s
2025-10-31T21:26:28.4175587Z  45150K .......... .......... .......... .                    100%  624M=1.0s
2025-10-31T21:26:28.4175965Z 
2025-10-31T21:26:28.4176272Z 2025-10-31 21:26:28 (43.1 MB/s) - written to stdout [46266273/46266273]
2025-10-31T21:26:28.4176688Z 
2025-10-31T21:26:28.4265795Z ##[group]Run ~/auto shipit -vv
2025-10-31T21:26:28.4266179Z [36;1m~/auto shipit -vv[0m
2025-10-31T21:26:28.4284636Z shell: /usr/bin/bash -e {0}
2025-10-31T21:26:28.4284953Z env:
2025-10-31T21:26:28.4285436Z   GH_TOKEN: ***
2025-10-31T21:26:28.4285720Z ##[endgroup]
2025-10-31T21:26:28.8358497Z ℹ  info      ENV: {
2025-10-31T21:26:28.8359072Z   isCi: true,
2025-10-31T21:26:28.8359631Z   name: 'GitHub Actions',
2025-10-31T21:26:28.8359991Z   service: 'github',
2025-10-31T21:26:28.8360329Z   commit: 'e0b7dba85e7a326c8ff4219cfeb526af9401c956',
2025-10-31T21:26:28.8360714Z   isPr: false,
2025-10-31T21:26:28.8360982Z   branch: 'main',
2025-10-31T21:26:28.8361255Z   prBranch: undefined,
2025-10-31T21:26:28.8361553Z   slug: 'neurobagel/query-tool',
2025-10-31T21:26:28.8361896Z   root: '/home/runner/work/query-tool/query-tool'
2025-10-31T21:26:28.8362235Z }
2025-10-31T21:26:28.8506761Z ✔  success   Using SEMVER labels: 
2025-10-31T21:26:28.8507442Z  Map(6) {
2025-10-31T21:26:28.8507876Z   'major' => [ 'pr-major-breaking' ],
2025-10-31T21:26:28.8508316Z   'minor' => [ 'pr-minor-breaking', 'pr-minor' ],
2025-10-31T21:26:28.8508845Z   'patch' => [ 'pr-patch', 'pr-bug-fix', 'pr-internal', 'pr-performance' ],
2025-10-31T21:26:28.8509505Z   'none' => [ 'pr-documentation', 'pr-tests', 'pr-dependencies' ],
2025-10-31T21:26:28.8509991Z   'skip' => [ 'skip-release' ],
2025-10-31T21:26:28.8510360Z   'release' => [ 'release' ]
2025-10-31T21:26:28.8510704Z }
2025-10-31T21:26:28.8520545Z ✔  success   Loaded `auto` with config: {
2025-10-31T21:26:28.8521339Z   onlyPublishWithReleaseLabel: true,
2025-10-31T21:26:28.8521989Z   noDefaultLabels: true,
2025-10-31T21:26:28.8522600Z   baseBranch: 'main',
2025-10-31T21:26:28.8523244Z   author: 'Neurobagel Bot <bot@neurobagel.org>',
2025-10-31T21:26:28.8523952Z   noVersionPrefix: false,
2025-10-31T21:26:28.8524520Z   plugins: [
2025-10-31T21:26:28.8525021Z     'git-tag',
2025-10-31T21:26:28.8525472Z     'released',
2025-10-31T21:26:28.8525824Z     'first-time-contributor',
2025-10-31T21:26:28.8526200Z     [ 'omit-commits', [Object] ],
2025-10-31T21:26:28.8526561Z     [ 'npm', [Object] ]
2025-10-31T21:26:28.8526865Z   ],
2025-10-31T21:26:28.8565292Z   ],
2025-10-31T21:26:28.8565548Z   prereleaseBranches: [ 'next' ],
2025-10-31T21:26:28.8565872Z   versionBranches: undefined
2025-10-31T21:26:28.8566159Z }
2025-10-31T21:26:28.8566816Z ℹ  info      Found plugin using: /snapshot/auto/packages/core/dist/plugins/filter-non-pull-request.js
2025-10-31T21:26:28.8567565Z ℹ  info      Found plugin using: /snapshot/auto/packages/core/dist/plugins/filter-non-pull-request.js
2025-10-31T21:26:28.8568202Z ℹ  info      Found plugin using: /snapshot/auto/plugins/git-tag/dist/index.js
2025-10-31T21:26:28.8568762Z ℹ  info      Found plugin using: /snapshot/auto/plugins/git-tag/dist/index.js
2025-10-31T21:26:28.8569330Z ℹ  info      Found plugin using: /snapshot/auto/plugins/released/dist/index.js
2025-10-31T21:26:28.8570077Z ℹ  info      Found plugin using: /snapshot/auto/plugins/released/dist/index.js
2025-10-31T21:26:28.8734516Z ℹ  info      Found plugin using: /snapshot/auto/plugins/first-time-contributor/dist/index.js
2025-10-31T21:26:28.8735766Z ℹ  info      Found plugin using: /snapshot/auto/plugins/first-time-contributor/dist/index.js
2025-10-31T21:26:28.8742247Z ℹ  info      Found plugin using: /snapshot/auto/plugins/omit-commits/dist/index.js
2025-10-31T21:26:28.8743425Z ℹ  info      Found plugin using: /snapshot/auto/plugins/omit-commits/dist/index.js
2025-10-31T21:26:28.8846286Z ℹ  info      Found plugin using: /snapshot/auto/plugins/npm/dist/index.js
2025-10-31T21:26:28.8847486Z ℹ  info      Found plugin using: /snapshot/auto/plugins/npm/dist/index.js
2025-10-31T21:26:28.8848503Z ℹ  info      Using Filter Non Pull Request Plugin...
2025-10-31T21:26:28.8849373Z ℹ  info      Using git-tag Plugin...
2025-10-31T21:26:28.8860333Z ℹ  info      Using released Plugin...
2025-10-31T21:26:28.8861117Z ℹ  info      Using first-time-contributor Plugin...
2025-10-31T21:26:28.8861822Z ℹ  info      Using omit-commits Plugin...
2025-10-31T21:26:28.8862386Z ℹ  info      Using npm Plugin...
2025-10-31T21:26:28.9112654Z https://github.com/neurobagel/query-tool
2025-10-31T21:26:28.9132549Z ℹ  info      Options contain repo information.
2025-10-31T21:26:28.9134890Z ℹ  info      Initializing GitHub API with:
2025-10-31T21:26:28.9135273Z  {
2025-10-31T21:26:28.9135516Z   owner: 'neurobagel',
2025-10-31T21:26:28.9135783Z   repo: 'query-tool',
2025-10-31T21:26:28.9136129Z   token: '[Token starting with ghs_]',
2025-10-31T21:26:28.9136676Z   agent: undefined,
2025-10-31T21:26:28.9137124Z   baseUrl: undefined,
2025-10-31T21:26:28.9137409Z   graphqlBaseUrl: undefined
2025-10-31T21:26:28.9137690Z }
2025-10-31T21:26:28.9138062Z ℹ  info      Initializing GitHub with: https://api.github.com
2025-10-31T21:26:28.9231078Z https://github.com/neurobagel/query-tool
2025-10-31T21:26:28.9237239Z ℹ  info      Getting project from GitHub
2025-10-31T21:26:36.2278803Z ℹ  info      Got response from query
2025-10-31T21:26:36.2279317Z  { repository: { object: { history: [Object] } } }
2025-10-31T21:26:36.2284377Z ℹ  info      Added release notes to changelog
2025-10-31T21:26:36.2301283Z ℹ  info      Added groups to changelog
2025-10-31T21:26:36.2309556Z ℹ  info      Added authors to changelog
2025-10-31T21:26:36.2310459Z ℹ  info      Successfully generated release notes.
2025-10-31T21:26:36.2311041Z ℹ  info      New Release Notes
2025-10-31T21:26:36.2311514Z  #### 🚀 Enhancements
2025-10-31T21:26:36.2311751Z 
2025-10-31T21:26:36.2312658Z - [ENH] Split query into dataset- and subject-level query [#651](https://github.com/neurobagel/query-tool/pull/651) ([@rmanaem](https://github.com/rmanaem))
2025-10-31T21:26:36.2313294Z 
2025-10-31T21:26:36.2313471Z #### 🏠 Internal
2025-10-31T21:26:36.2313665Z 
2025-10-31T21:26:36.2314159Z - [FIX] Manually bumped `package.json` version [#657](https://github.com/neurobagel/query-tool/pull/657) ([@rmanaem](https://github.com/rmanaem))
2025-10-31T21:26:36.2315173Z - [ENH] Add npm plugin to auto [#656](https://github.com/neurobagel/query-tool/pull/656) ([@surchs](https://github.com/surchs))
2025-10-31T21:26:36.2315684Z 
2025-10-31T21:26:36.2315826Z #### Authors: 2
2025-10-31T21:26:36.2316020Z 
2025-10-31T21:26:36.2316223Z - Arman Jahanpour ([@rmanaem](https://github.com/rmanaem))
2025-10-31T21:26:36.2316656Z - Sebastian Urchs ([@surchs](https://github.com/surchs))
2025-10-31T21:26:36.2375849Z v0.12.0
2025-10-31T21:26:36.2381491Z ℹ  info      Adding new changes to CHANGELOG.md.
2025-10-31T21:26:36.2387000Z ℹ  info      Calculating SEMVER bump using:
2025-10-31T21:26:36.2387716Z  {
2025-10-31T21:26:36.2388387Z   labels: [ [ 'release', 'pr-internal' ], [ 'pr-minor' ], [ 'pr-internal' ] ],
2025-10-31T21:26:36.2389206Z   versionLabels: Map(6) {
2025-10-31T21:26:36.2390014Z     'major' => [ 'pr-major-breaking' ],
2025-10-31T21:26:36.2390698Z     'minor' => [ 'pr-minor-breaking', 'pr-minor' ],
2025-10-31T21:26:36.2391507Z     'patch' => [ 'pr-patch', 'pr-bug-fix', 'pr-internal', 'pr-performance' ],
2025-10-31T21:26:36.2392121Z     'none' => [
2025-10-31T21:26:36.2392513Z       'pr-documentation',
2025-10-31T21:26:36.2392933Z       'pr-tests',
2025-10-31T21:26:36.2393623Z       'pr-dependencies',
2025-10-31T21:26:36.2394172Z       'released',
2025-10-31T21:26:36.2394670Z       'prerelease'
2025-10-31T21:26:36.2395066Z     ],
2025-10-31T21:26:36.2395474Z     'skip' => [ 'skip-release' ],
2025-10-31T21:26:36.2395965Z     'release' => [ 'release' ]
2025-10-31T21:26:36.2396440Z   },
2025-10-31T21:26:36.2397003Z   options: { onlyPublishWithReleaseLabel: true }
2025-10-31T21:26:36.2397675Z }
2025-10-31T21:26:36.2398245Z ✔  success   Calculated SEMVER bump: minor
2025-10-31T21:26:36.2398943Z ℹ  info      Calculated next version to be: 0.13.0
2025-10-31T21:26:36.2399700Z ℹ  info      Old changelog exists, prepending changes.
2025-10-31T21:26:36.2404777Z ℹ  info      Wrote new changelog to filesystem.
2025-10-31T21:26:36.2591417Z [main e44d158] Update CHANGELOG.md [skip ci]
2025-10-31T21:26:36.2592210Z  1 file changed, 18 insertions(+)
2025-10-31T21:26:36.2596461Z ℹ  info      Committed new changelog.
2025-10-31T21:26:36.2657119Z ℹ  info      Calling version hook
2025-10-31T21:26:36.2713907Z v0.12.0
2025-10-31T21:26:36.2717704Z ℹ  info      Tagging new tag: v0.12.0 => v0.13.0
2025-10-31T21:26:38.1297711Z npm error code E404
2025-10-31T21:26:38.1298910Z npm error 404 Not Found - GET https://registry.npmjs.org/react-query-tool - Not found
2025-10-31T21:26:38.1300711Z npm error 404
2025-10-31T21:26:38.1301259Z npm error 404  'react-query-tool@*' is not in this registry.
2025-10-31T21:26:38.1301960Z npm error 404
2025-10-31T21:26:38.1302459Z npm error 404 Note that you can also install from a
2025-10-31T21:26:38.1303149Z npm error 404 tarball, folder, http url, or git url.
2025-10-31T21:26:38.1310110Z npm error A complete log of this run can be found in: /home/runner/.npm/_logs/2025-10-31T21_26_36_551Z-debug-0.log
2025-10-31T21:26:38.2101683Z npm verbose cli /usr/local/bin/node /usr/local/bin/npm
2025-10-31T21:26:38.2102498Z npm info using npm@10.8.2
2025-10-31T21:26:38.2104197Z npm info using node@v20.19.5
2025-10-31T21:26:38.2104800Z npm silly config load:file:/usr/local/lib/node_modules/npm/npmrc
2025-10-31T21:26:38.2105408Z npm silly config load:file:/home/runner/work/query-tool/query-tool/.npmrc
2025-10-31T21:26:38.2105949Z npm silly config load:file:/home/runner/.npmrc
2025-10-31T21:26:38.2106396Z npm silly config load:file:/usr/local/etc/npmrc
2025-10-31T21:26:38.2123038Z npm verbose title npm version 0.13.0
2025-10-31T21:26:38.2124294Z npm verbose argv "version" "0.13.0" "--no-commit-hooks" "--message" "Bump version to: %s [skip ci]" "--loglevel" "silly"
2025-10-31T21:26:38.2127191Z npm verbose logfile logs-max:10 dir:/home/runner/.npm/_logs/2025-10-31T21_26_38_186Z-
2025-10-31T21:26:38.2136448Z npm verbose logfile /home/runner/.npm/_logs/2025-10-31T21_26_38_186Z-debug-0.log
2025-10-31T21:26:38.2834975Z npm silly logfile done cleaning log files
2025-10-31T21:26:38.3305568Z npm verbose stack Error: An unknown git error occurred
2025-10-31T21:26:38.3306442Z npm verbose stack     at makeError (/usr/local/lib/node_modules/npm/node_modules/@npmcli/git/lib/make-error.js:28:13)
2025-10-31T21:26:38.3307330Z npm verbose stack     at /usr/local/lib/node_modules/npm/node_modules/@npmcli/git/lib/spawn.js:37:26
2025-10-31T21:26:38.3308126Z npm verbose stack     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2025-10-31T21:26:38.3309686Z npm verbose stack     at async module.exports (/usr/local/lib/node_modules/npm/node_modules/libnpmversion/lib/version.js:117:5)
2025-10-31T21:26:38.3311349Z npm verbose stack     at async Version.change (/usr/local/lib/node_modules/npm/lib/commands/version.js:75:21)
2025-10-31T21:26:38.3312659Z npm verbose stack     at async Npm.exec (/usr/local/lib/node_modules/npm/lib/npm.js:207:9)
2025-10-31T21:26:38.3313375Z npm verbose stack     at async module.exports (/usr/local/lib/node_modules/npm/lib/cli/entry.js:74:5)
2025-10-31T21:26:38.3313955Z npm error code 128
2025-10-31T21:26:38.3314247Z npm error An unknown git error occurred
2025-10-31T21:26:38.3314926Z npm error command /usr/bin/git --no-replace-objects tag -m Bump version to: 0.13.0 [skip ci] v0.13.0
2025-10-31T21:26:38.3315450Z npm error fatal: tag 'v0.13.0' already exists
2025-10-31T21:26:38.3320381Z npm verbose cwd /home/runner/work/query-tool/query-tool
2025-10-31T21:26:38.3321238Z npm verbose os Linux 6.11.0-1018-azure
2025-10-31T21:26:38.3321973Z npm verbose node v20.19.5
2025-10-31T21:26:38.3322606Z npm verbose npm  v10.8.2
2025-10-31T21:26:38.3323401Z npm verbose exit 128
2025-10-31T21:26:38.3324593Z npm verbose code 128
2025-10-31T21:26:38.3326082Z npm error A complete log of this run can be found in: /home/runner/.npm/_logs/2025-10-31T21_26_38_186Z-debug-0.log
2025-10-31T21:26:38.3362580Z Error: Running command 'npm' with args [version, 0.13.0, --no-commit-hooks, -m, 'Bump version to: %s [skip ci]', --loglevel, silly] failed
2025-10-31T21:26:38.3363364Z 
2025-10-31T21:26:38.3363633Z npm verbose cli /usr/local/bin/node /usr/local/bin/npm
2025-10-31T21:26:38.3364230Z npm info using npm@10.8.2
2025-10-31T21:26:38.3364693Z npm info using node@v20.19.5
2025-10-31T21:26:38.3365307Z npm silly config load:file:/usr/local/lib/node_modules/npm/npmrc
2025-10-31T21:26:38.3366156Z npm silly config load:file:/home/runner/work/query-tool/query-tool/.npmrc
2025-10-31T21:26:38.3366772Z npm silly config load:file:/home/runner/.npmrc
2025-10-31T21:26:38.3367135Z npm silly config load:file:/usr/local/etc/npmrc
2025-10-31T21:26:38.3367493Z npm verbose title npm version 0.13.0
2025-10-31T21:26:38.3368112Z npm verbose argv "version" "0.13.0" "--no-commit-hooks" "--message" "Bump version to: %s [skip ci]" "--loglevel" "silly"
2025-10-31T21:26:38.3368802Z npm verbose logfile logs-max:10 dir:/home/runner/.npm/_logs/2025-10-31T21_26_38_186Z-
2025-10-31T21:26:38.3369342Z npm verbose logfile /home/runner/.npm/_logs/2025-10-31T21_26_38_186Z-debug-0.log
2025-10-31T21:26:38.3369999Z npm silly logfile done cleaning log files
2025-10-31T21:26:38.3370372Z npm verbose stack Error: An unknown git error occurred
2025-10-31T21:26:38.3370964Z npm verbose stack     at makeError (/usr/local/lib/node_modules/npm/node_modules/@npmcli/git/lib/make-error.js:28:13)
2025-10-31T21:26:38.3371666Z npm verbose stack     at /usr/local/lib/node_modules/npm/node_modules/@npmcli/git/lib/spawn.js:37:26
2025-10-31T21:26:38.3372336Z npm verbose stack     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
2025-10-31T21:26:38.3373081Z npm verbose stack     at async module.exports (/usr/local/lib/node_modules/npm/node_modules/libnpmversion/lib/version.js:117:5)
2025-10-31T21:26:38.3374020Z npm verbose stack     at async Version.change (/usr/local/lib/node_modules/npm/lib/commands/version.js:75:21)
2025-10-31T21:26:38.3374652Z npm verbose stack     at async Npm.exec (/usr/local/lib/node_modules/npm/lib/npm.js:207:9)
2025-10-31T21:26:38.3375255Z npm verbose stack     at async module.exports (/usr/local/lib/node_modules/npm/lib/cli/entry.js:74:5)
2025-10-31T21:26:38.3375738Z npm error code 128
2025-10-31T21:26:38.3376040Z npm error An unknown git error occurred
2025-10-31T21:26:38.3376562Z npm error command /usr/bin/git --no-replace-objects tag -m Bump version to: 0.13.0 [skip ci] v0.13.0
2025-10-31T21:26:38.3377072Z npm error fatal: tag 'v0.13.0' already exists
2025-10-31T21:26:38.3377464Z npm verbose cwd /home/runner/work/query-tool/query-tool
2025-10-31T21:26:38.3377842Z npm verbose os Linux 6.11.0-1018-azure
2025-10-31T21:26:38.3378176Z npm verbose node v20.19.5
2025-10-31T21:26:38.3378475Z npm verbose npm  v10.8.2
2025-10-31T21:26:38.3378767Z npm verbose exit 128
2025-10-31T21:26:38.3379042Z npm verbose code 128
2025-10-31T21:26:38.3379691Z npm error A complete log of this run can be found in: /home/runner/.npm/_logs/2025-10-31T21_26_38_186Z-debug-0.log
2025-10-31T21:26:38.3380075Z 
2025-10-31T21:26:38.3380373Z     at ChildProcess.<anonymous> (/snapshot/auto/packages/core/src/utils/exec-promise.ts:76:23)
2025-10-31T21:26:38.3380882Z     at ChildProcess.emit (node:events:527:28)
2025-10-31T21:26:38.3381244Z     at ChildProcess.emit (node:domain:475:12)
2025-10-31T21:26:38.3381803Z     at Process.ChildProcess._handle.onexit (node:internal/child_process:291:12)Error
2025-10-31T21:26:38.3382414Z     at Object.execPromise (/snapshot/auto/packages/core/src/utils/exec-promise.ts:17:20)
2025-10-31T21:26:38.3382908Z     at /snapshot/auto/plugins/npm/src/index.ts:1056:15
2025-10-31T21:26:38.3383369Z     at processTicksAndRejections (node:internal/process/task_queues:96:5)
2025-10-31T21:26:38.3433829Z ##[error]Process completed with exit code 1.
2025-10-31T21:26:38.3598835Z Post job cleanup.
2025-10-31T21:26:38.4327956Z [command]/usr/bin/git version
2025-10-31T21:26:38.4358903Z git version 2.51.2
2025-10-31T21:26:38.4391702Z Temporarily overriding HOME='/home/runner/work/_temp/1e522c98-5ccd-439f-8e6f-899178f465d8' before making global git config changes
2025-10-31T21:26:38.4393110Z Adding repository directory to the temporary git global config as a safe directory
2025-10-31T21:26:38.4397450Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/query-tool/query-tool
2025-10-31T21:26:38.4427083Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2025-10-31T21:26:38.4451542Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2025-10-31T21:26:38.4619085Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2025-10-31T21:26:38.4635772Z http.https://github.com/.extraheader
2025-10-31T21:26:38.4643858Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2025-10-31T21:26:38.4668270Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2025-10-31T21:26:38.5003898Z Post job cleanup.
2025-10-31T21:26:38.8870217Z Token revoked
2025-10-31T21:26:38.9550802Z Cleaning up orphan processes
```
</details>

### Error message

```
npm error fatal: tag 'v0.13.0' already exists
```

### Environment

GH workflow runner

### How to reproduce

_No response_

### Anything else?

The problem is most likely because 

- we use the auto `git-tag` plugin: https://github.com/neurobagel/query-tool/blob/e0b7dba85e7a326c8ff4219cfeb526af9401c956/.autorc#L8
- we now also use the `npm` plugin

Both of these plugins want to create a git tag for the new release:

- https://github.com/intuit/auto/tree/main/plugins/git-tag
- https://docs.npmjs.com/cli/v6/commands/npm-version#description

And this triggers a race. We need to tell one of them to not make a git-tag. 

One option is to have a npm config file `.npmrc` and set

```
git-tag-version=false
```
https://docs.npmjs.com/cli/v6/commands/npm-version#git-tag-version

There is some chance that this will also prevent the npm plugin from commiting the package.json file - which would be bad. Guess we'll have to test it

--- a/.autorc
+++ b/.autorc
@@ -19,7 +19,10 @@
         [
             "npm",
             {
-                "setRcToken": false
+                "setRcToken": false,
+                "exact": true,
+                "commitNextVersion": true,
+                "publishFolder": "dist"
             }
         ]
     ],

--- a/.npmrc
+++ b/.npmrc
@@ -1,0 +1,1 @@
+git-tag-version=false

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "react-query-tool",
   "private": true,
-  "version": "v0.12.0",
+  "version": "0.12.0",
   "type": "module",
   "scripts": {
     "dev": "vite",

